radix-engine/src/blueprints/resource/package.rs

```diff
@@ -1376,133 +1504,304 @@ impl ResourceManagerNativePackage {
                     NonFungibleResourceManagerBlueprint::get_non_fungible(receiver, input.id, api)?;
                 Ok(IndexedScryptoValue::from_typed(&rtn))
             }
-            VAULT_LOCK_FEE_IDENT => {
+            FUNGIBLE_VAULT_LOCK_FEE_IDENT => {
```

Comment:
It's a bit weird to me that all the blueprints in the resource native package share this invoke list which means they all have to be unique idents, but I guess we're all aware it's a bit weird and will change it at some point 🤷

Reply:
Sure, but I'd assume it makes sense to first match on Blueprint, then delegate to a separate method on each blueprint which matches method name? :).

Also, is `FUNGIBLE_VAULT_LOCK_FEE_IDENT = ("fungible_vault", "lock_fee")`? Wasn't obvious from the code :)

Reply:
At the code level, there is no notion of "blueprint", just code. The match on `(Blueprint, method)` already occurred before this big match statement, ie `(Blueprint, method)` resolves to some unique string method in the package.

Reply:
Ahh gotcha 👍. Feels a little weird to me to go from `(Blueprint, method)` to Blueprint_Method, but not a big deal on the grand schema of things,